### PR TITLE
CloudServicesHost Incorrect version in folder, csproj references v7

### PR DESCRIPTION
**Pull request by ramonsmits**

### CODE REVIEW

**Comment by ramonsmits** [comment]
@DavidBoike should this sample be completely removed?

File: Snippets/CloudServicesHost/CloudServicesHost.sln [hunk 1]
```diff
@@ -2,7 +2,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29728.190
 MinimumVisualStudioVersion = 15.0.26730.12
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CloudServicesHost_8", "CloudServicesHost_8\CloudServicesHost_8.csproj", "{8BC068F0-831B-4D8F-B9F5-A67B1435FEB8}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CloudServicesHost_7", "CloudServicesHost_7\CloudServicesHost_8.csproj", "{8BC068F0-831B-4D8F-B9F5-A67B1435FEB8}"
```
Comment (awright18) [inline Snippets/CloudServicesHost/CloudServicesHost.sln:5]:
```suggestion
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CloudServicesHost_7", "CloudServicesHost_7\CloudServicesHost_7.csproj", "{8BC068F0-831B-4D8F-B9F5-A67B1435FEB8}"
```

**Comment by DavidBoike** [comment]
We definitely don’t need a V8 sample for something we are only shipping a “tombstone package” for in V8.

**Comment by ramonsmits** [comment]
Its failing with:

> System.Exception: Could not find page reference for /nservicebus/hosting/cloud-services-host

I searched for `cloud-services-host` but no files contain that text.

@DavidBoike any idea?

**Pull request merged**